agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -594,6 +595,37 @@ void backhaul_manager::after_select(bool timeout)
         discovery_timestamp = now;
         send_1905_topology_discovery_message();
     }
+
+    // Each platform must send Topology Discovery message every 60 seconds to its first circle
+    // 1905.1 neighbors.
+    // Iterate on each of known 1905.1 neighbors and check if we have received in the last
+    // 60 seconds a Topology Discovery message from it. If not, remove this neighbor from our list
+    // and send a Topology Notification message.
+    bool neighbors_list_changed = false;
+    for (auto it = m_1905_neighbor_devices.begin(); it != m_1905_neighbor_devices.end();) {
+        const auto &last_topology_discovery = it->second;
+        if (last_topology_discovery + std::chrono::seconds(DISCOVERY_NEIGHBOUR_REMOVAL_TIMEOUT) <
+            std::chrono::steady_clock::now()) {
+            const auto &device_al_mac = it->first;
+            LOG(INFO) << "Removed 1905.1 device " << device_al_mac << " from neighbors list";
+            it                     = m_1905_neighbor_devices.erase(it);
+            neighbors_list_changed = true;
+            continue;
+        }
+        it++;
```

Comment:
I find this easier to read:
```suggestion
        } else {
            it++;
        }
```

(nitpick and personal preference only anyway)